src/__tests__/footer.test.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-undef */
+import { render, screen, cleanup, act } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Footer from "../components/Footer";
+import React from "react";
+import App from "../App";
+import "@testing-library/jest-dom";
+import userEvent from "@testing-library/user-event";
+import { configureStore } from "@reduxjs/toolkit";
+import rootReducer from "../reducers/index";
+import { Provider } from "react-redux";
+import firebase from "firebase/compat/app";
+import "firebase/compat/firestore";
+import "firebase/compat/auth";
+
+window.scrollTo = jest.fn();
```

Comment:
Do we need this ?

Reply:
We need configstore,provider and rootReducer